### DISCUSSION TITLE
MongoDB

### DISCUSSION THREAD

**Discussion by daVinci13** [tobychui/zoraxy#159] [Q&A] [answered]
How to point sub.domain.com to mongodb using tcp?

**Comment by tobychui** [comment]
No, you can't do that as it is out of the scope of Zoraxy and here is the [reasons](https://stackoverflow.com/questions/46490499/get-subdomain-from-tcp-socket). You need to go to your DNS service provider and create a A record for your subdomain DNS and point it to your machine with mongodb installed.

**Reply by daVinci13** [reply]
Database is in a docker container, other services that use http are working like a charm, however i can't connect to MongoDB at all.

**Reply by tobychui** [reply]
If you are unable to connect to MongoDB, then it is a docker networking issue (connection not working between Zoraxy in your host OS and your dockerized app). You might wanna seek support from the docker community.

**Reply by daVinci13** [reply]
Mongo requires stream proxy, as said other services work as intended when proxied through Zoraxy.

**Reply by tobychui** [reply]
I guess by stream proxy you mean TCP proxy. Try the TCP Proxy function with transport mode to see if it works?

**Reply by daVinci13** [reply]
Unless i'm doing something wrong, it does not work... I opened a port on docker, and set Zoraxy to listen on that port, and transfer trafic to MongoDB container (10.10.10.15:27017).

**Reply by tobychui** [reply, marked as answer]
I just have another quick test and I don't see any issues regarding the TCP proxy function. These kind of issues are mostly caused by networking within your setup, so I guess you need to figure out this issue yourself.

**Reply by daVinci13** [reply]
Thank you for your time and assistance. I will mark this discussion as answered. Keep up the excellent work! ⭐⭐⭐⭐⭐
